pest/src/position.rs

```diff
@@ -91,17 +89,6 @@ impl<'i> Position<'i> {
     }
 
     /// Returns the line - and column number pair of the current `Position`.
-    ///
```

Comment:
Why remove the example here? `Position::line_col` is useful to the user.

Reply:
I'll come up with a better example.